torch/fx/graph.py

```diff
@@ -627,11 +628,52 @@ def delete_unused_values(user : Node):
             else:
                 body.append('\n')
 
+        def preprocess_values(a : Argument):
+            """
+            This function preprocesses values with `args`/`kwargs` just before
+            pretty printing
+            """
+            if isinstance(a, tuple) and hasattr(a, '_fields'):
+                # Annoyingly, Python NamedTuples repr() with an unqualified
```

Comment:
I don't know if supporting named tuples in arguments is worth this complexity. Can we just capture something simpler than the named tuple and convert named tuples to it? I am pretty sure this isn't the first patch fixing named tuples and seems to be adding more complexity than the value it adds.

```
class Tuple:
   def __init__(self, **kwargs):
     self.__dict__.update(kwargs)
   def __repr__(self):
       return f"torch.fx.Tuple(...)"
   def __iter__(self):
        return iter(self.values())
   def __getitem__(self, i):
        return self.values()[i]
```

Reply:
This patch is primarily aimed at the case where a `NamedTuple` is the return from a given `Module`, which is an extremely common pattern. I don't think returning a generic tuple is going to work, since surrounding code that calls the module may expect the object returned to be indexable by name